### PR TITLE
First pass at Chameleon Cloud ITB backfill containers (FD#66938)

### PR DESCRIPTION
**Pull request by brianhlin**
Hi @zhenz, here's the first pass at the backfill container registration. Could you look it over and make suggestions for incorrect descriptions (particularly `Description` and `AddressLine1` in `SITE.yaml`)?

### CODE REVIEW

File: topology/University of Chicago/Chameleon/SITE.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# LongName is the expanded name of the site
+LongName: Chameleon Cloud at the University of Chicago
+# Description is a brief description of your site
+Description: Computer Science building of the University of XYZ
```
Comment (zhenz) [inline topology/University of Chicago/Chameleon/SITE.yaml:4]:
```suggestion
Description: Theory and Computing Sciences Building at Argonne National Laboratory
```

File: topology/University of Chicago/Chameleon/SITE.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+# If you have an up-to-date local git clone, fill ID with the output from `bin/next_site_id`
+# Otherwise, leave it blank and we will fill in the appropriate value for you.
+ID: 10294
+
+# AddressLine1 is the street address of the site
+AddressLine1: 1234 Fifth St
```
Comment (zhenz) [inline topology/University of Chicago/Chameleon/SITE.yaml:11]: 9700 S. Cass Ave

Comment (zhenz) [inline topology/University of Chicago/Chameleon/SITE.yaml:11]:
```suggestion
AddressLine1: 9700 S. Cass Ave
```

File: topology/University of Chicago/Chameleon/SITE.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+### AddressLine2 (optional) is the second line of the street address
+# AddressLine2: Room 101
+# City is the city the site is located in
+City: Chicago
```
Comment (zhenz) [inline topology/University of Chicago/Chameleon/SITE.yaml:15]:
```suggestion
City: Lemont
```

File: topology/University of Chicago/Chameleon/SITE.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+# Country is the country the site is located in
+Country: USA
+### State (optional) is the state or province the site is located in
+# State: Wisconsin
```
Comment (zhenz) [inline topology/University of Chicago/Chameleon/SITE.yaml:20]:
```suggestion
# State: Illinois
```

File: topology/University of Chicago/Chameleon/SITE.yaml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+### Zipcode (optional) is the zipcode/postal code of the site
+# Zipcode: "01234"
```
Comment (zhenz) [inline topology/University of Chicago/Chameleon/SITE.yaml:22]:
```suggestion
Zipcode: "60439"
```

File: topology/University of Chicago/Chameleon/SITE.yaml [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+# Latitude is the latitude of the site (positive values are in the northern hemisphere)
+Latitude: 41.790270
```
Comment (zhenz) [inline topology/University of Chicago/Chameleon/SITE.yaml:25]:
```suggestion
Latitude: 41.71833
```

File: topology/University of Chicago/Chameleon/SITE.yaml [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+# Longitude is the longitude of the site (positive values are in the eastern hemisphere)
+Longitude: -87.601670
```
Comment (zhenz) [inline topology/University of Chicago/Chameleon/SITE.yaml:27]:
```suggestion
Longitude: -87.98237
```

**Comment by brianhlin** [comment]
@edquist can I get a quick review? I can't approve my own PRs :)

**Review by edquist — APPROVED** [review]
Looks fine to me, assuming the check run comes back ok

**Pull request merged**